test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/Common/Command/Shell/DockerRunCommand.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.Common;
+
+/* Shell Command responsible for creating running the docker container 
+* containing function app images for particular language.
+*/
+public class DockerRunCommand : ShellCommand
+{
+	public DockerRunCommand(BrokerType brokerType, Language language)
+	{
+		cmd = BuildDockerStartCmd(brokerType, language);
+	}
+
+	private string BuildDockerStartCmd(BrokerType brokerType, Language language)
+	{
+		//Starts the list with docker run and port specific to language
+		var cmdList = new List<string>
+		{
+			Constants.DOCKER_RUN, Constants.DOCKER_PORT_FLAG,
+			$"{Constants.BrokerLanguagePortMapping[new Tuple<BrokerType, Language>(brokerType, language)]}{Constants.COLON_7071}"
+		};
+
+		//Adding Provider Specific variables 
+		if (BrokerType.CONFLUENT == brokerType)
+		{
+			cmdList.Add(Constants.DOCKER_ENVVAR_FLAG);
```

Comment:
nit pick : this can be break down into small function for confluent & eventhub

Reply:
Sure. We can take this up as with the improvement task.